datafusion/src/logical_plan/plan.rs

```diff
@@ -378,11 +392,12 @@ impl LogicalPlan {
                 _ => vec![],
             },
             LogicalPlan::Window { window_expr, .. } => window_expr.clone(),
-            LogicalPlan::Aggregate {
-                group_expr,
-                aggr_expr,
-                ..
-            } => group_expr.iter().chain(aggr_expr.iter()).cloned().collect(),
+            LogicalPlan::Aggregate(aggregate) => aggregate
```

Comment:
Same opinion like above, if we keep consistent like before, we can make less changes.

Reply:
please align the code style using
```
LogicalPlan::Aggregate(aggregate {
  field1，
  field2
})
```